README.md

```diff
@@ -623,6 +623,7 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
   * [Virushee](https://virushee.com/) — Privacy-oriented file/data scanning powered by hybrid heuristic and AI-assisted engine. Possible to use internal dynamic sandbox analysis. Limited to 50MB per file upload
   * [Escape GraphQL Quickscan](https://escape.tech/) - One-click security scan of your GraphQL endpoints. Free, no login required.
   * [FingerprintJS Browser Fingerprinting](https://www.fingerprint.com) - FingerprintJS is a browser fingerprinting library that queries browser attributes and computes a hashed visitor identifier from them. Unlike cookies and local storage, a fingerprint stays the same in incognito/private mode and even when browser data is purged.
+    * Free trial version of Pro has 20,000 Requests per month
```

Comment:
please add it on the same line like other examples seen here.  While you're there change "free trial" to "free tier" as this is not a site for trials (your free plan isnt a trial so best avoid describing it as a trial)